DS/Assignment2/mycollectionclasses/CollectionsMain.java

```diff
@@ -0,0 +1,60 @@
+package mycollectionclasses;
+
+import java.util.Scanner;
+
+import mycollectionclasses.college.counseling.Councellor;
+
+public class CollectionsMain {
+
+    public static void main(String[] args) {
+	Stacks();
+	infixToPostfix();
+	Queues();
+	councellor();
+    }
+
+    public static void Queues() {
+	Queue queue = new Queue();
+	queue.enqueue(1);
+	queue.enqueue(2);
+	queue.enqueue(3);
+	queue.display();
+	queue.dequeue();
+	queue.dequeue();
+	System.out.println(queue.getFront());
+	queue.makeEmpty();
+	queue.display();
+    }
+
+    public static void Stacks() {
+	Stack stack = new Stack();
+	stack.push(1);
+	stack.push(2);
+	stack.push(10);
+	stack.push(11);
+	stack.push(12);
+	stack.push(13);
+	stack.display();
+	System.out.println(">" + stack.size());
+	stack.pop();
+	stack.display();
+	System.out.println(">" + stack.size());
+    }
+
+    public static void infixToPostfix() {
+	InfixToPostFix infixToPostFix = new InfixToPostFix();
+	Scanner scanner = new Scanner(System.in);
+	System.out.print("Enter Infix : ");
+	String infix = scanner.next();
```

Comment:
use anonymous object if reference not use more than one place like : 
String infix = new Scanner(System.in).next();